test/cmp_ctx_test.c

```diff
@@ -13,6 +13,10 @@
 
 #include <openssl/x509_vfy.h>
 
+static X509 *test_cert;
+
+#define X509_new() X509_dup(test_cert)
+
```

Comment:
This looks like a leftover from experiments.

Reply:
No, it is needed.

Reply:
Actually it would not be necessary to change all the other X509_new() calls below, but I did it anyway for clarity.

Reply:
Ah, meanwhile I see that due to the generic definition of the call within `DEFINE_SET_TEST()` unfortunately it is needed,
though a redefinition of the function `X509_new()` is kind of dirty - I suggest adding a comment why it is needed.